api/v1beta1/disruption_webhook.go

```diff
@@ -95,7 +95,8 @@ var _ webhook.Validator = &Disruption{}
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (r *Disruption) ValidateCreate() error {
-	logger.Debugw("validating created disruption", "instance", r.Name, "namespace", r.Namespace)
+	logger := logger.With("instance", r.Name, "namespace", r.Namespace)
```

Comment:
```suggestion
	logger := logger.With("instance", r.Name, "namespace", r.Namespace)
```

Should we continue to align our logging keys,`disruptionName` and `disruptionNamespace` are used in the reconcile loop and injector, could we use such keys here also?

Maybe we could have a logger wrapper/builder that return a logger from a disruption with expected keys so we spread/reuse it whenever needed.

Reply:
It depends, are you talking about logs or metrics :trollface: 

https://github.com/DataDog/chaos-controller/blob/4cfb483c2af30fe139ccee39f1fc59b93a7d7ed2/controllers/disruption_controller.go#L101

Reply:
Metrics yes! But having everything aligned as well would be nice too 😅

Reply:
Yes definitely, and here I would complain about the fact metric tags are "lowercased" and it's more readable to me to have words separated by underscore :trollface: hence `disruption_name` and `disruption_namespace` are my favs.